README.md

```diff
@@ -1,48 +1,74 @@
-repo-openio-sds
-===============
+[![Build Status](https://travis-ci.org/open-io/ansible-role-openio-repository.svg?branch=master)](https://travis-ci.org/open-io/ansible-role-openio-repository)
+# Ansible role `repository`
 
-Ansible role that configure OpenIO SDS repositories.
+An Ansible role for setup OpenIO's repositories. Specifically, the responsibilities of this role are to:
 
-Requirements
-------------
+- Setup key and repository for an OpenIO product and a specific distribution
```

Comment:
"specific linux distribution" ?

Reply:
nope specific distribution of the product :) :vulgarize: